specification/index.md

```diff
@@ -1,48 +1,42 @@
-E-ARK CSIP Table Of Contents
-============================
-
-Version: 2.0.0
-
-May 31, 2019
 
 Contents
 --------
-- [1 Introduction](#1-introduction)
-	- [1.1 The Common Specification for Information Packages and OAIS](#11-the-common-specification-for-information-packages-and-oais)
-	- [1.2 The Common Specification for Information Packages and Content Information Type Specifications](#12-the-common-specification-for-information-packages-and-content-information-type-specifications)
-	- [1.3 Common Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#13-common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
-	- [1.4. Relation to other documents](#14-relation-to-other-documents)
+- [troduction](#introduction)
+	- [he Common Specification for Information Packages and OAIS](#the-common-specification-for-information-packages-and-oais)
+	- [he Common Specification for Information Packages and Content Information Type Specifications](#the-common-specification-for-information-packages-and-content-information-type-specifications)
+	- [ommon Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
```

Comment:
Here i get lost in the change. Is letters missing?

Reply:
This file's to go so don't worry about it.